conf/variant/qemu-x86-64_nogfx/local.conf.sample

```diff
@@ -0,0 +1,3 @@
+MACHINE = "qemux86-64"
+
+require conf/variant/common/local.conf
```

Comment:
I would prefer it if the MACHINE setting came after the require line, so that we could potentially overwrite variables from it.